api/lib/validate.js

```diff
@@ -166,8 +178,95 @@ function validateOrderbook(orderbook) {
   }
 }
 
+function validateSettings(settings) {
+  if (typeof settings !== 'object') {
+    throw error('Invalid parameter: settings');
```

Comment:
How should the client handle them? Currently, it looks like this: https://github.com/ripple/ripple-rest/blob/develop/server/error-handler.js

Maybe we should move this to `api/`, and the `server/` gets an error object with information it can use to serve in the response